main.go

```diff
@@ -1,17 +1,30 @@
 package main
 
 import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-
+	"context"
+	"flag"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"log"
 	"terraform-provider-powerstore/powerstore"
 )
 
+var (
+	version string = "dev"
```

Comment:
shouldn't we make this as command line argument or may be a environment variable